picard/mbxml.py

```diff
@@ -107,7 +126,17 @@ def _relations_to_metadata(relation_lists, m):
             for relation in relation_list.relation:
                 if relation.type == 'performance':
                     performance_to_metadata(relation, m)
+                    if ('work' not in relation.children
+                        or 'id' not in relation.work[0].attribs
+                        ):
+                        log.warning('MBXML: Recording work skipped - no target work MBID present')
```

Comment:
I am in two minds about this sort of warning message. On the one hand, MB XML is generally reliable, but on the other hand if we start to get these sorts of inconsistencies we would want to know about them.

But I am not adding these consistently across Picard in this PR, so happy to remove in order to be consistent. Please advise.